modules/javafx.controls/src/main/java/javafx/scene/control/Tooltip.java

```diff
@@ -170,6 +171,16 @@ public Tooltip(String text) {
         getStyleClass().setAll("tooltip");
     }
 
+    @Override
+    protected void show() {
```

Comment:
Another idea I was trying that worked fine is to have a new method inside `PopupWindow`, which we can call rather than `show()`:

```
protected final void applySceneStylesFromOwner(Window owner) {
    Scene sceneValue = getScene();
    final Scene ownerScene = getRootWindow(owner).getScene();
    if (ownerScene != null) {
        if (ownerScene.getUserAgentStylesheet() != null) {
            sceneValue.setUserAgentStylesheet(ownerScene.getUserAgentStylesheet());
        }
        sceneValue.getStylesheets().setAll(ownerScene.getStylesheets());
        if (sceneValue.getCursor() == null) {
            sceneValue.setCursor(ownerScene.getCursor());
        }
    }
}
```

The `show()` method inside `PopupWindow` is also doing excactly the code above, thats why I ended up calling it. 
But we can also think about just calling that code directly and not `show`, we just need to extract that part of the code and create a `protected` method with it.